src/layers/invertible_layer_actnorm.jl

```diff
@@ -59,7 +59,7 @@ end
 # 2-3D Foward pass: Input X, Output Y
 function forward(X::AbstractArray{T, N}, AN::ActNorm; logdet=nothing) where {T, N}
     isnothing(logdet) ? logdet = (AN.logdet && ~AN.is_reversed) : logdet = logdet
-    inds = [i!=(N-1) ? 1 : (:) for i=1:N]
+    inds = [i!=(N-1) ? 1 : Colon() for i=1:N]
```

Comment:
Is this related to style or are there other benefits?

Reply:
Is easy to forget parenthesis and leads to unreadable error. `Colon` is safer IMO